pkg/asset/installconfig/openstack/validation/machinepool_test.go

```diff
@@ -137,9 +137,13 @@ func TestOpenStackMachinepoolValidation(t *testing.T) {
 				mp.FlavorName = notExistFlavor
 				return mp
 			}(),
-			cloudInfo:      validMpoolCloudInfo(),
+			cloudInfo: func() *CloudInfo {
+				ci := validMpoolCloudInfo()
+				ci.Flavors[notExistFlavor] = nil
+				return ci
+			}(),
 			expectedError:  true,
-			expectedErrMsg: "controlPlane.platform.openstack.type: Not found: \"non-existant-flavor\"",
+			expectedErrMsg: "controlPlane.platform.openstack.flavorName: Not found: \"non-existant-flavor\"",
```

Comment:
it is called "type", not "flavorName": https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L428

let's keep it as it was before